receiver/jmxreceiver/factory.go

```diff
@@ -41,6 +41,7 @@ func createDefaultConfig() config.Receiver {
 	return &Config{
 		ReceiverSettings:   config.NewReceiverSettings(config.NewID(typeStr)),
 		JARPath:            "/opt/opentelemetry-java-contrib-jmx-metrics.jar",
+		Properties:         make(map[string]string),
```

Comment:
If you look at the code example above just adding 
```suggestion
		Properties:         map[string]string{"org.slf4j.simpleLogger.defaultLogLevel": "info"},
```

would work in the same way because of how maps are merged (hadn't realized until now, sorry). I think that's simpler, wdyt?

Reply:
Yup, that's a great recommendation, and helps ensure the Properties always have the full & correct value after unmarshalling. Added a test to validate overriding it, looks good.